doc/source/setup/security.rst

```diff
@@ -6,6 +6,11 @@ Secure the installation
 Close ports
 ===========
 
+CrateDB
+-------
+
+.. todo:: Outline how to protect CrateDB's public listening ports.
```

Comment:
I know CrateDB has special handling for that purpose, in order to do [DWIM](https://en.wikipedia.org/wiki/DWIM). Do you have a suggestion how to phrase a corresponding statement here?